.circleci/config.yml

```diff
@@ -0,0 +1,32 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: circleci/node:8.9.3
+
+    working_directory: ~/repo
+
+    steps:
+      - checkout
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+          - v1-dependencies-{{ checksum "package.json" }}
+          # fallback to using the latest cache if no exact match is found
+          - v1-dependencies-
+
+      - run: npm install
+      - save_cache:
+          paths:
+            - node_modules
+          key: v1-dependencies-{{ checksum "package.json" }}
+      - run:
+          name: Lint JS
+          command: npm run lint
+      - run:
+          name: Run tests
+          command: npm run test
```

Comment:
we should probably use `parallelism: 4` and `--maxWorkers=4` for [jest to make use of it's parallel functionality](https://vgpena.github.io/jest-circleci/). There's a [good example in the kiln config](https://github.com/clay/clay-kiln/blob/master/.circleci/config.yml).

Also, are we checking code coverage (via coveralls/etc) when we run tests?

Reply:
good idea, will add